src/test/java/com/exasol/drivers/ExasolDriverManagerIT.java

```diff
@@ -3,50 +3,131 @@
+        final ExasolDriverManager exasolDriverManager = EXASOL.getDriverManager();
+        final DatabaseDriver driver = JdbcDriver.builder(DERBY_DRIVER_NAME) //
+                .prefix("jdbc:derby:") //
+                .sourceFile(driverPath) //
+                .mainClass(derbyDriver.getClass().getCanonicalName()) //
+                .enableSecurityManager(false) //
+                .build();
+        exasolDriverManager.install(driver);
+        final String expectedContent = "from Derby";
+        prepareDerbyInMemoryTableWithContent(expectedContent, hostIP);
+        try (final Connection exasolConnection = EXASOL.createConnection();
+             final Statement statement = exasolConnection.createStatement();
+             final ResultSet result = statement.executeQuery("IMPORT FROM JDBC " //
+                     + "AT 'jdbc:derby://" + hostIP + ":" + DERBY_PORT + "/test' " //
+                     + "USER '" + DERBY_USER + "' IDENTIFIED BY 'open sesame'"
+                     + "STATEMENT 'SELECT * FROM T'")) {
+            result.next();
+            assertThat(result.getString(1), equalTo(expectedContent));
+        }
+    }
+
+    private static Path getPathToDriverJarFile(final Driver derbyDriver) {
+        return Path.of(derbyDriver.getClass().getProtectionDomain().getCodeSource().getLocation().getFile());
+    }
+
+    private void prepareDerbyInMemoryTableWithContent(final String content, final String hostIP) throws SQLException {
+        try (final Connection derbyConnection =
+                     DriverManager.getConnection("jdbc:derby://" + hostIP + ":" + DERBY_PORT + "/test;create=true;");
+             final Statement statement = derbyConnection.createStatement()) {
+            statement.execute("CREATE TABLE T(C VARCHAR(40))");
+            statement.execute("INSERT INTO T VALUES ('" + content + "')");
+        }
+    }
+
+    private static void startDerbyServer(final Path derbyHomeDir, final String hostIP) throws Exception {
+        System.setProperty("derby.system.home", derbyHomeDir.toString());
+        System.setProperty("derby.user."+ DERBY_USER, DERBY_PASSWORD);
+        derbyServer = new NetworkServerControl(InetAddress.getByName(hostIP), DERBY_PORT);
+        derbyServer.start(new PrintWriter(System.out));
+    }
+
+    @SuppressWarnings("java:S2925") // We need to wait for the server to come up with "sleep" in a loop.
+    private static void waitForDerbyServerToAcceptConnections() throws InterruptedException {
+        for (int attempts = 0; attempts < 20; ++attempts) {
+            try {
+                derbyServer.ping();
+                return;
+            }
+            catch(final Exception exception) {
+                Thread.sleep(500);
+            }
+        }
+        throw new IllegalStateException("Derby database server required for integration tests did not start up.");
+    }
+
+    private static void shutdownDerbyServer() throws Exception {
```

Comment:
Call this in afterEach()

Reply:
Added try ... finally, so that only the test that is using the derby server does that.